bundles/org.openhab.binding.tuya/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -61,7 +61,7 @@
 	<thing-type id="tuyaDevice" extensible="color,switch,dimmer,number,string,ir-code">
 		<label>Generic Tuya Device</label>
 		<description>A generic Tuya device. Can be extended with channels.</description>
-
+		<semantic-equipment-tag>ControlDevice</semantic-equipment-tag>
```

Comment:
Tuya devices aren't necessarily controllable (for example, some are just temperature/humidity sensors). So this should either just be `Equipment`, or not have a tag at all-- unless/until someone goes deeper in the binding and dynamically calculates a better tag depending on what actually exists on the device.

Reply:
Ok, I will remove the tag from the xml.

You could consider adding `updateThing(editThing().withSemanticEquipmentTag("whatever").build())` in `handler.initialize()`..